datadog-monitors-monitor-handler/src/datadog_monitors_monitor/handlers.py

```diff
@@ -0,0 +1,291 @@
+import logging
+from typing import Any, MutableMapping, Optional
+
+from cloudformation_cli_python_lib import (
+    Action,
+    HandlerErrorCode,
+    OperationStatus,
+    ProgressEvent,
+    Resource,
+    SessionProxy,
+)
+from datadog_api_client.v1 import ApiException
+from datadog_api_client.v1.api.monitors_api import MonitorsApi
+from datadog_api_client.v1.model.monitor import Monitor as ApiMonitor
+from datadog_api_client.v1.model.monitor_options import MonitorOptions as ApiMonitorOptions
+from datadog_api_client.v1.model.monitor_threshold_window_options import \
+    MonitorThresholdWindowOptions as ApiMonitorThresholdWindows
+from datadog_api_client.v1.model.monitor_thresholds import MonitorThresholds as ApiMonitorThresholds
+from datadog_api_client.v1.model.monitor_type import MonitorType as ApiMonitorType
+from datadog_api_client.v1.model.monitor_update_request import MonitorUpdateRequest as ApiMonitorUpdateRequest
+from datadog_cloudformation_common.api_clients import v1_client
+
+from .models import Creator, MonitorOptions, MonitorThresholdWindows, \
+    MonitorThresholds, \
+    ResourceHandlerRequest, \
+    ResourceModel
+from .version import __version__
+
+# Use this logger to forward log messages to CloudWatch Logs.
+LOG = logging.getLogger(__name__)
+TYPE_NAME = "Datadog::Monitors::Monitor"
+TELEMETRY_TYPE_NAME = "monitors-monitor"
+
+resource = Resource(TYPE_NAME, ResourceModel)
+test_entrypoint = resource.test_entrypoint
+
+
+@resource.handler(Action.READ)
+def read_handler(
+        session: Optional[SessionProxy],
+        request: ResourceHandlerRequest,
+        callback_context: MutableMapping[str, Any],
+) -> ProgressEvent:
+    LOG.info("Starting %s Read Handler", TYPE_NAME)
+    model = request.desiredResourceState
+    with v1_client(
+            model.DatadogCredentials.ApiKey,
+            model.DatadogCredentials.ApplicationKey,
+            model.DatadogCredentials.ApiURL or "https://api.datadoghq.com",
```

Comment:
Is it possible to set a default in the schema?

Reply:
Sure, let's do it in a next PR, in case we do it in the common package too